docs/UserGuide.adoc

```diff
@@ -380,9 +381,15 @@ Examples:
 
 === Sort food consumption or exercise record
 
+Sorts the records of food consumption or exercise workout based on time of record or calorie,
+in either ascending or descending order.
+
+format: `sortX -c/-t -a/-d`
```

Comment:
There is some ambiguity in the draft UG: do we sort records of food intake and exercise workout separately or all together? That probably depends on how the data is stored.